spec/hangman_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require './lib/hangman'
```

Comment:
Normally the `lib` directory is added to project's `$LOAD_PATHS` (this is where ruby looks for `require`d files). 

Therefore, this should work:

```suggestion
require 'hangman'
```